templates/manage_certs.py

```diff
@@ -32,7 +33,7 @@ def get_all_domains(config):
             line = line.strip()
             if line and not line.startswith("#"):
                 domain, backend = line.split(None, 1)
-                domains[domain] = backend
+                domains[domain.lower()] = backend
```

Comment:
@antoviaque Yup, all agreed, in particular about the importance of having the infrastructure in place.